.github/workflows/flamegraph.yml

```
@@ -0,0 +1,36 @@
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    - uses: actions/checkout@v3
+    - name: Install Dependencies
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:69: Quote this to prevent word splitting

(shellcheck)